packageRpm.nix

```diff
@@ -44,15 +74,16 @@ in rec {
         mkdir rpmbuild
         cd rpmbuild
         mkdir SPECS
+        mkdir SOURCES
         cp ${writeSpecFile} SPECS/${project}.spec
+        cp ${sourceArchive} SOURCES/${sourceArchive.name}
+        ls SOURCES
```

Comment:
Is it only for logging purposes or is the output used by the rest of the script somehow?

Reply:
Oh, I forgot to remove this line